backend/resolvers/mutations/show.js

```diff
@@ -9,6 +9,17 @@ export function createShow (_, args, req) {
   return Show.create(args.input)
 }
 
+export function updateShow (_, args, req) {
```

Comment:
Do we want to allow updating all the fields of a show right now? I feel like this could add in some edge cases w/ dates. Perhaps make a mutation just for finalizing since that's only what we need right now.

Reply:
I think we should keep this mutation and instead just not allow for updating dates in a new ShowUpdate type (for now).